Makefile

```diff
@@ -134,6 +134,8 @@ clean-sql-files:
 
 clean-extra:
 	@rm -f src/*~
+	@rm -f src/*.d
```

Comment:
@erimatnor is this change okay? I kept seeing left over .o & .d files when the source files would get renamed/removed